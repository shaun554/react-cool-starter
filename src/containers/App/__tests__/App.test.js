// import React from 'react';
// import { shallow } from 'enzyme';
//
// import App from '../index';
//
// test('<App />', () => {
//   const routes = [{}];
//   const wrapper = shallow(<App routes={routes} />);
// });
