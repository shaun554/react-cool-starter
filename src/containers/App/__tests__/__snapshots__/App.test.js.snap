// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <img
      alt="Logo"
      role="presentation"
      src="IMAGE_MOCK"
    />
    <h1>
      React Cool Starter
    </h1>
  </div>
  <hr />
  <div
    className={undefined}
  >
    <p>
      Loading...
    </p>
  </div>
</div>
`;
